lib/image_optim/worker/svgo.rb

```diff
@@ -4,11 +4,17 @@ class ImageOptim
   class Worker
     # https://github.com/svg/svgo
     class Svgo < Worker
+      DISABLE_PLUGINS_OPTION =
+      option(:disable_plugins, [], Array, 'List of plugins to disable'){ |v| v }
```

Comment:
No need to provide `Array` as an argument as type will be guessed from `[]`.
Better to use `Array(v).map(&:to_s)` in the proc to have no problems with single values and arrays with non string values.